app/models/consultation_response.rb

```diff
@@ -20,6 +20,10 @@ def access_limited_object
 
   delegate :organisations, to: :parent_attachable
 
+  delegate :lead_organisations, to: :parent_attachable
+
+  delegate :supporting_organisations, to: :parent_attachable
```

Comment:
Is there a test that can be added to ensure CallForEvidenceResponse and ConsultationResponse delegate these methods? As removing these lines would presumably cause some issues down the line!

Reply:
Good idea. Have added tests for this.